core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -49,7 +49,7 @@
     </button>
     <div class="oppia-audio-visulaizer">
       <div class="oppia-mp3-converting" ng-if="recorder.status.isConverting">
-        Converting to mp3<loading-dots></loading-dots>
+        Converting to mp3 <loading-dots></loading-dots>
```

Comment:
Remove this space. (We don't use space before loading-dots)

Reply:
solved this